### PR TITLE
Update HDFS Package for EA2

### PR DESCRIPTION
**Pull request by elingg**
@ConnorDoyle, Please take a look at your earliest convenience, thx! I will also create a similar PR for master once this is merged.

### CODE REVIEW

**Comment by ConnorDoyle** [comment]
For these changes, let's target `master` and then sync to `ea2`.  Trying to make sure we don't get out of sync or lose any important changes in `master`.

File: repo/packages/H/hdfs/0/config.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "type": "string"
       },
       "default": [
-        "https://downloads.mesosphere.io/hdfs/dcos/0.1.0/hdfs-mesos-0.1.0.tgz"
+        "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/dcos-ea/0.1.0/hdfs-mesos-0.1.0.tgz"
```
Comment (ConnorDoyle) [inline repo/packages/H/hdfs/0/config.json]: Did the software change, or is it just configuration?  If there were source code modifications we should rev the version instead (doesn't look like a snapshot, so assuming the tag is immutable).

**Comment by elingg** [comment]
Sure, @connorDoyle. Let me do that.

**Comment by ConnorDoyle** [comment]
Thanks!

**Pull request closed without merging**